### DISCUSSION TITLE
Help with Python import

### DISCUSSION THREAD

**Discussion by adelka-m** [zippy84/vtkbool#78] [Q&A]
Hi,
first of all, thank you for the library. 

I would like to ask for help, as I have troubles with importing the library into python.
I think I installed everything correctly:
- "vtkbool is configured as a library:" the so file is here: `/xxx/vtkboolbuild/build/lib/python3.10/site-packages/vtkbool/vtkBool.so`
- "Python 3 is installed with header files:" `/usr/include/python3.10/Python.h`
- "VTK itself is wrapped to Python:" `import vtk` works

But if I run 
```
import sys
sys.path.append('/xxx/vtkboolbuild/build/')
from vtkBool import vtkPolyDataBooleanFilter
```
gives ModuleNotFoundError: No module named 'vtkBool'

Where did it go wrong?

Thank you!

**Comment by zippy84** [comment]
I think it must be something like:

sys.path.extend(['/xxx/vtkboolbuild/build/lib/python3.10/site-packages/vtkbool'])

**Comment by adelka-m** [comment]
Thanks for the swift answer. I have changed the code to the following and it runs:

```
import sys
sys.path.append('/xxx/vtkboolbuild/build/lib/python3.10/site-packages/vtkbool')
sys.path.append('/opt/VTK-9.3.0/build/lib/python3.10/site-packages/')
```

You also have to uninstall the pip vtk module from you environment and use the build vtk on your system. Otherwise it gave me error like here [https://github.com/zippy84/vtkbool/issues/61](https://github.com/zippy84/vtkbool/issues/61).


(Maybe is usefull for future users.)
